.github/workflows/test-unit.yml

```diff
@@ -22,9 +22,11 @@ jobs:
       - uses: actions/checkout@v2
       - run: git fetch origin master --depth=10
       - run: git fetch origin ${{ github.ref }} --depth=10
-      - uses: actions/setup-node@v1
       - run: yarn install
       - run: yarn run build
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
```

Comment:
For some reason, unit tests were failing during `ncc` so I used the same pattern as `test-integration-cli.yml` and `test-integration-dev.yml` and that seemed to fix it.

Its still a mystery why it ever worked before.

Reply:
Seems weird that we install node _after_ running `yarn`, right?

Reply:
Maybe we need to add `with` to every `run` step?

Take a look at the previous commit before this `with` was moved: https://github.com/vercel/vercel/runs/918955588#step:7:192